src/ClientNet462/Program.cs

```diff
@@ -0,0 +1,148 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Client
+{
+    internal class Program
+    {
+        static async Task Main(string[] args)
+        {
+            try
+            {
+                using (var cts = new CancellationTokenSource())
```

Comment:
```suggestion
                using var cts = new CancellationTokenSource();
```

Saves 1 indent for example.

Reply:
Not supported in .NET 4.6.2

Reply:
(Or more specifically, the version of C# required for that syntax isn't supported by .NET 4.6.2, or something like that.)

Reply:
🤦